psi4/src/psi4/libmints/orthog.cc

```diff
@@ -137,34 +140,56 @@ void BasisSetOrthogonalization::compute_inverse() {
 }
 
 SharedMatrix BasisSetOrthogonalization::overlap_inverse() {
-    auto Sinv = std::make_shared<Matrix>("Inverse Transformation", X_->rowspi(), X_->rowspi());
+    auto Sinv = std::make_shared<Matrix>("Inverse overlap matrix", X_->rowspi(), X_->rowspi());
     Sinv->gemm(false, true, 1.0, X_, X_, 0.0);
     return Sinv;
 }
 
 void BasisSetOrthogonalization::compute_symmetric_orthog() {
+    /*
+      Susi Lehtola Jan 16 2020
+
+      SO basis functions aren't normalized, which causes X to be
+      asymmetric at the end (see unroll_normalization function). It
+      might be possible to achieve a symmetric X even with the use of
+      balancing i.e. renormalization of the basis set, but this would
+      require tracking the dependence of the eigenvectors and
+      eigenvalues on the scaling transform.
+
+      If one omits the normalization with the commented code below, X
+      becomes symmetric. However, the symmetricity of X or the lack of
+      it makes no difference at the end, since the orbitals anyway arise
+      from diagonalizing the Fock matrix or orbital optimization, and
+      any code in Psi4 should also work with canonical orthogonalization
+      which is asymmetric by force.
+    */
```

Comment:
Thanks for this note - it will be really helpful in a few years when we inevitably revisit this code and wonder what's going on